templates/frontend/objects/article_details.tpl

```diff
@@ -166,15 +173,18 @@
 								{if $author->getLocalizedAffiliation()}
 									<div class="article-author-affiliation">
 										{$author->getLocalizedAffiliation()|escape}
+										{if $author->getData('rorId')}
+											<a href="{$author->getData('rorId')|escape}">{$rorIdIcon}</a>
```

Comment:
This is subject to change based on @gaziyucel's active work on RORs and multiple affiliations.

Reply:
Yes, I'm keeping an eye on that issue and can either wait to merge the theme updates or do another PR once that has been merged.